sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/ExpressionLanguageInjection.kt

```diff
@@ -171,4 +171,23 @@ object ExpressionLanguageInjection {
         val expr = arguments[0] as? String ?: return
         Jazzer.guideTowardsContainment(expr, SPRING_EXPRESSION_LANGUAGE_ATTACK, hookId)
     }
+
+    @MethodHook(
+        type = HookType.BEFORE,
+        targetClassName = "org.apache.commons.jexl2.JexlEngine",
+        targetMethod = "createExpression",
```

Comment:
Would the [createScript](https://commons.apache.org/proper/commons-jexl/javadocs/apidocs-2.1/org/apache/commons/jexl2/JexlEngine.html#createScript(java.lang.String)) functions also be interesting to hook?